src/index.ts

```diff
@@ -89,7 +85,7 @@ export async function resolve(
   }
 
   // Let Node.js handle all other specifiers.
-  return defaultResolve(specifier, context, defaultResolve);
+  return defaultResolve(specifier, { ...context, parentURL }, defaultResolve);
```

Comment:
Although there isn't anything else on context on most Node.JS versions, I want to future proof this for the chance that there is/will be extra data in context.